integration/resource-routes-test.ts

```diff
@@ -208,7 +208,9 @@ test.describe("loader in an app", async () => {
     expect(await res.text()).toEqual("Partial");
   });
 
-  test("should handle objects returned from resource routes", async ({
+  // TODO: Matt and I chatted about this recently, revisit that chat and
```

Comment:
@brophdawg11 I'm disabling this and we can update / re-enable it after your PR goes in.

Reply:
yeah this should work after https://github.com/remix-run/remix/pull/9349 is brought over - wanted to do that after this was merged to avoid conflicts